database/seeds/RolesSeeder.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use App\Role;
+
+class RolesSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        Role::create([
+            'name' => 'admin'
+        ]);
+        Role::create([
+            'name' => 'collegist'
+        ]);
+        Role::create([
+            'name' => 'tenant'
+        ]);
+        Role::create([
+            'name' => 'workshop-administrator'
+        ]);
```

Comment:
We should not do this in a seeder instead of in a migration because we will not run the seeds in production.